code/__DEFINES/_flags/_flags.dm

```diff
@@ -26,32 +26,30 @@ GLOBAL_LIST_INIT(bitflags, list(
 
 //FLAGS BITMASK
 ///This flag is what recursive_hear_check() uses to determine wether to add an item to the hearer list or not.
-#define HEAR_1						(1<<3)
+#define HEAR_1						(1<<0)
 ///Projectiles will use default chance-based ricochet handling on things with this.
-#define DEFAULT_RICOCHET_1			(1<<4)
+#define DEFAULT_RICOCHET_1			(1<<1)
 ///Conducts electricity (metal etc.).
-#define CONDUCT_1					(1<<5)
+#define CONDUCT_1					(1<<2)
 ///For machines and structures that should not break into parts, eg, holodeck stuff.
-#define NODECONSTRUCT_1				(1<<7)
-///Atom queued to SSoverlay.
-#define OVERLAY_QUEUED_1			(1<<8)
+#define NODECONSTRUCT_1				(1<<3)
 ///Item has priority to check when entering or leaving.
-#define ON_BORDER_1					(1<<9)
+#define ON_BORDER_1					(1<<4)
 ///Whether or not this atom shows screentips when hovered over
-#define NO_SCREENTIPS_1				(1<<10)
+#define NO_SCREENTIPS_1				(1<<5)
 ///Prevent clicking things below it on the same turf eg. doors/ fulltile windows.
-#define PREVENT_CLICK_UNDER_1		(1<<11)
-#define HOLOGRAM_1					(1<<12)
+#define PREVENT_CLICK_UNDER_1		(1<<6)
+#define HOLOGRAM_1					(1<<7)
 ///Prevents mobs from getting chainshocked by teslas and the supermatter.
-#define SHOCKED_1 					(1<<13)
+#define SHOCKED_1 					(1<<8)
 ///Whether /atom/Initialize() has already run for the object.
-#define INITIALIZED_1				(1<<14)
+#define INITIALIZED_1				(1<<9)
 ///was this spawned by an admin? used for stat tracking stuff.
-#define ADMIN_SPAWNED_1				(1<<15)
+#define ADMIN_SPAWNED_1				(1<<10)
 /// should not get harmed if this gets caught by an explosion?
-#define PREVENT_CONTENTS_EXPLOSION_1 (1<<16)
+#define PREVENT_CONTENTS_EXPLOSION_1 (1<<11)
 /// Early returns mob.face_atom()
-#define BLOCK_FACE_ATOM_1			(1<<17)
+#define BLOCK_FACE_ATOM_1			(1<<12)
```

Comment:
These are all tied to `flags_1`, an atom proc, as are the below turf flags. Wouldn't this cause conflicts where, for instance, anything tagged with `HEAR_1` is also `NO_JAUNT_1`, and anything with `UNUSED_RESERVATION_TURF_1` is also `DEFAULT_RICOCHET_1`?

Reply:
Oh, it might conflict? I need to check some stuff.